docs/api.rst

```diff
@@ -110,3 +110,126 @@ Methods
+   * - Calculate climatologies?
+     - ``Dataset.temporal.climatology("VAR_KEY", freq=<"season"|"month"|"day">)``, subset results for individual seasons, months, or days
+     - ``cdutil.SEASONALCYCLE.climatology()``, ``cdutil.ANNUALCYCLE.climatology()``, ``cdutil.<DJF|MAM|JJA|SON>.climatology()``, ``cdutil.<JAN|FEB|...|DEC>.climatology()``
+   * - Calculate climatological departures?
+     - ``Dataset.temporal.departures("VAR_KEY", freq=<"season"|"month"|"day">)``, subset results for individual seasons, months, or days
+     - ``cdutil.SEASONALCYCLE.departures()``, ``cdutil.ANNUALCYCLE.departures()``, ``cdutil.<DJF|MAM|JJA|SON>.departures()``, ``cdutil.<JAN|FEB|...|DEC>.departures()``
+   * - Center time coordinates using time bounds?
+     - ``Dataset.temporal.center_times()``
+     - N/A
+   * - Regrid horizontally?
+     - ``Dataset.regridder.horizontal_regrid2()``, ``Dataset.regridder.horizontal_xesmf()``
+     - ``cdms2.regrid2()``
+
+
+``cf_xarray`` for interpreting CF-compliant attributes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``xcdat`` leverages ``cf_xarray`` for interpreting CF-compliant attributes (``.attrs``) present on ``xarray.DataArray`` or ``xarray.Dataset`` objects.
+
+.. pull-quote::
+  One powerful feature of ``cf_xarray`` is the ability to refer to named dimensions by standard ``axis`` or ``coordinate`` names in Dataset or DataArray methods.
+
+  -- https://cf-xarray.readthedocs.io/en/latest/coord_axes.html#axes-and-coordinates:
+
+.. code-block:: python
+
+  from cf_xarray.datasets import airds
+
+  airds.cf
+
+  Coordinates:
+  - CF Axes: * X: ['lon']
+            * Y: ['lat']
+            * T: ['time']
+              Z: n/a
+
+  - CF Coordinates: * longitude: ['lon']
+                    * latitude: ['lat']
+                    * time: ['time']
+                      vertical: n/a
+
+  - Cell Measures:   area: ['cell_area']
+                    volume: n/a
+
+  - Standard Names: * latitude: ['lat']
+                    * longitude: ['lon']
+                    * time: ['time']
+
+  - Bounds:   n/a
+
+  Data Variables:
+  - Cell Measures:   area, volume: n/a
+
+  - Standard Names:   air_temperature: ['air']
+
+  - Bounds:   n/a
+
+
+.. list-table::
+   :widths: 20 40 40
+   :header-rows: 1
+
+   * - How do I...
+     - cf_xarray
+     - CDAT
+   * - Get latitude coordinates (Y axis)?
+     - ``Dataset.cf["Y"]``, ``Dataset.cf["lat"]``, or ``Dataset.cf["latitude"]``
+     - ``TransientVariable.getLatitude()``
+   * - Get longitude coordinates (X axis)?
+     - ``Dataset.cf["X"]``, ``Dataset.cf["lon"]``, or ``Dataset.cf["longitude"]``
+     - ``TransientVariable.getLongitude()``
+   * - Get time coordinates (T axis)?
+     - ``Dataset.cf["T"]`` or ``Dataset.cf["time"]``
+     - ``TransientVariable.getTime()``
+   * - Get vertical coordinates (Z axis)?
+     - ``Dataset.cf["Z"]``
+     - ``TransientVariable.getLevel()``
```

Comment:
Hi @durack1, let me know how this looks based on your suggestion from a previous comment.

Reply:
On second thought, this information might be best placed in another repo (e.g., `xcdat_utils`) since the APIs here aren't from xCDAT.